content/rs/networking/cluster-dns.md

```diff
@@ -0,0 +1,116 @@
+---
+Title: Configure Cluster DNS
+linkTitle: Configure Cluster DNS
+description: Configure DNS to communicate between nodes in your cluster.
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: [
+        /rs/administering/installing-upgrading/configuring/cluster-name-dns-connection-management/,
+        /rs/installing-upgrading/configuring/cluster-name-dns-connection-management/,
+        /rs/networking/cluster-dns/,
+]
+
+---
+
+By default, Redis Enterprise Software deployments use DNS to communicate between nodes.  You can also use the [Discovery Service]({{< relref "/rs/databases/configure/discovery-service.md" >}}), which uses IP addresses to connect and complies with the [Redis Sentinel API](https://redis.io/topics/sentinel) supported by open source Redis.
+
+Each node in a Redis Enterprise cluster includes a small DNS server to manage internal functions, such as high availability, automatic failover, automatic migration, and so on.
+Nodes should only run the DNS server included with the software.  Running additional DNS servers can lead to unexpected behavior.
+
+## Cluster name and connection management
+
+Whether you're administering Redis Enterprise Software or accessing databases, there are two ways to connect:
+
+- URL-based connections - URL-based connections use DNS to resolve the fully qualified cluster domain name (FQDN).  This means that DNS records might need to be updated when topology changes, such as adding (or removing) nodes from the cluster.  
+
+    Because apps and other clients connections rely on the URL (rather than the address), they do not need to be modified when topology changes.  
+
+- IP-based connections - IP-based connections do not require DNS setup, as they rely on the underlying TCP/IP addresses.  As long as topology changes do not change the address of the cluster nodes, no configuration changes are needed, DNS or otherwise.  
+
+    However, changes to IP addresses (or changes to IP address access) impact all connections to the node, including apps and clients.  IP address changes can therefore be unpredictable or time-consuming.
+
+## URL-based connections
+
+The fully qualified domain name (FQDN) is the unique cluster identifier that enables clients to connect to [the different components]({{< relref "/rs/concepts/_index.md" >}}) of Redis Enterprise Software.
+The FQDN is a crucial component of the high-availability mechanism because it's used internally to enable and implement automatic and transparent failover of nodes, databases shards, and endpoints.
```

Comment:
Should "databases shards" be changed to either "databases, shards" or "database shards" here?